contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -25,12 +27,30 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
     IVoltzVault internal _vault;
     uint256[] internal _pullExistentials;
     IMarginEngine internal _marginEngine;
+    IPeriphery internal _periphery;
+    IVAMM internal _vamm;
     IVoltzVault.TickRange internal _currentPosition;
 
     // MUTABLE PARAMS
 
     uint256 _lastSignal;
     uint256 _lastLeverage;
+    uint256 _k_unwind_parameter; // parameter for k*leverage (for unwinding so this needs to be sent to the contract vault but not used in the strategy vault)
+    uint256 _sigmaWad; // y (standard deviation parameter in wad 10^18)
+    int256 _max_possible_lower_bound_wad; // should be in fixed rate
+
+    int24 _logProximity; // x (closeness parameter in wad 10^18) in log base 1.0001
+    int24 _currentTick;
+    int24 _tickLower;
+    int24 _tickUpper;
```

Comment:
I don't see why we store current tick, lower tick or upper tick here. We should retrieve them directly from the vault.

Reply:
Ticks are being retrieved from vault and these were removed